tests/EvmAcceptanceTests/test/precompiles/EvmPrecompiles.js

```diff
@@ -0,0 +1,28 @@
+const {expect} = require("chai");
+const {web3} = require("hardhat");
+const web3_helper = require("../../helper/Web3Helper");
+const general_helper = require("../../helper/GeneralHelper");
+
+describe("Precompile tests with web3.js", function () {
+  describe("When recover function is used", function () {
```

Comment:
seems like this can be merged into the `it` block. we use `describe` when there are at least 2 related tests to that `describe`. If you can add more tests to `When recover function is used` I'm OK with it.

Reply:
Thanks, done.